README.md

```diff
@@ -1 +1,113 @@
-# To-Do-list
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>To Do List</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+**[To Do List]** is a project which adds/removes tasks to the list.<br />
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ - HTML 
+ - CSS
+ - JAVASCRIPT (ES6)
+ - WEBPACK
+
+### Key Features <a name="key-features"></a>
+
+- **[Used ES6 Javascript]**
+- **[Used Webpack]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Here I will attach Live Demo link when the project will be ready.
+
```

Comment:
- [ ] Kindly remove this header as suggested ealier if there is no live demo at the moment. This will make your README file to be more engaging and professional. 👍